gcc/config/riscv/riscv-buildins-zacas.def

```diff
@@ -0,0 +1,6 @@
+// ZACAS
+RISCV_BUILTIN(amocas_si_si, "amocas32", RISCV_BUILTIN_ZACAS, RISCV_VOID_FTYPE_SI_SI_VOID_PTR, zacas_amocassi32),
+RISCV_BUILTIN(amocas_si_di, "amocas32", RISCV_BUILTIN_ZACAS, RISCV_VOID_FTYPE_SI_SI_VOID_PTR, zacas_amocassi64),
```

Comment:
the builtin name is si_di but type use SI_SI_PTR

Reply:
rename to amocas32si ...